### DISCUSSION TITLE
React Native [Expo] : Logged In View doesn't seem to open up

### DISCUSSION THREAD

**Discussion by anshumantekriwal** [Web3Auth/Web3Auth#1237] [Help]
Hello,
I was trying to integrate the W3A React Native SDK with the Expo workflow. I referred to the example on Expo in your `examples` repository and wanted to test it out. I replaced the `clientId` and proceeded to run it on my local environment. 

However, while it did login, instead of redirecting to the `loggedInView` or logging something onto the console, it just takes me to the localhost environment with the `key` [ eg. http://localhost:19006/web3auth#eyJwcml2... ] and logs an error onto the console for the previous tab open. Logged in view doesn't open.  

What workaround might there be to this?

Thank You!

**Comment by yashovardhan** [comment]
Hey Ansh,

Thanks for letting us know about the issue you're facing. I'm still not sure about the exact error you're facing, some console logs/ screenshots will be helpful here.

I might also tell you that you need to whitelist the URL of your expo instance in the Web3Auth dashboard for the login to work.

**Reply by anshumantekriwal** [reply]
> Hey Ansh,
> 
> Thanks for letting us know about the issue you're facing. I'm still not sure about the exact error you're facing, some console logs/ screenshots will be helpful here.
> 
> I might also tell you that you need to whitelist the URL of your expo instance in the Web3Auth dashboard for the login to work.

Hey! 

There is no error as such on the console. it simply says `Login flow interrupted.`
As for whitelisting the instance,I didn't whitelist the local instance as it said that I didn't need to do it for development servers. I'll do that and get back to you!
